Calendar/View Controllers/DetailsViewController.m

```diff
@@ -0,0 +1,116 @@
+//
+//  DetailsViewController.m
+//  Calendar
+//
+//  Created by Angelina Zhang on 7/13/22.
+//
+
+#import "DetailsViewController.h"
+#import "DetailsView.h"
+#import "ComposeViewController.h"
+
+@interface DetailsViewController () <DetailsViewDelegate, ComposeViewControllerDelegate>
+
+@property (strong, nonatomic) IBOutlet DetailsView *detailsView;
+
+@end
+
+@implementation DetailsViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    self.detailsView.delegate = self;
+    [self updateDetailsView];
```

Comment:
so minor semantics but I might call this `configureDetailsView` or `populateDetailsView` or `decorateDetailsView`... I can't put my finger on why update sounds wrong to me. I think because update implies a change and this is an initial state. Nothing is actually changing (technically there are other calls before this but this is basically the start).